docs/Documentation/Development/VSCode/index.md

```diff
@@ -77,3 +77,33 @@ Now use the Remote-SSH extension in VS Code to `Connect to Host...` and use the
 
 This should open a new VS Code window that will connect to the compute node automatically. You may begin browsing your home directory and editing files in the VS Code window.
 
+## Jupyter in VS Code on Kestrel
+
+### Setting Up VS Code
+To begin, proceed to VSCode and install these additional extensions, if you do not already have them: [Python](https://marketplace.visualstudio.com/items?itemName=ms-python.python) and [Jupyter](https://marketplace.visualstudio.com/items?itemName=ms-toolsai.jupyter).
+
+### Setting Up Conda Environment
+In addition, you will also need to set up a Python Environment on Kestrel. At a minimum, you must perform the following commands:
+
+```
+module load anaconda3
+conda create -n myJupEnv
+conda activate myJupEnv
+conda install Jupyter
+```
+
+### Intializing Environment
+
+Before using VS Code to connect to a compute, the compute node has to be allocated. Log onto Kestrel via a terminal and allocate 2 nodes by using this line:
+```
+salloc -A <projectname> -t 00:30:00 --partition=shared
+```
+
+### Running the Code
+
+To begin, open VS Code and hit F1 to run a command. Step through the process of connecting to a compute node. Once connected to the compute node, go to the left side of the screen and open your home directory on Kestrel (also, you may trust your own home directory, but do not click the check box). 
+
+Then, open the "cupyAndIpyparallel.ipynb" file in VS Code. Go to the top right corner and select the Python Kernel created earlier. VS Code may prompt for installing some Python packages; please allow these or the process will not work.
```

Comment:
cupyAndIpyparallel.ipynb is no longer being used as an example, so it shouldn't be referenced here. 

This section and the one above can be condensed. Link to the example above of how to start a Job and connect vscode, then explain that you can open a notebook and need to select the environment created earlier